cypress/e2e/tests/migration/migration-waves/filter.test.ts

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright © 2021 the Konveyor Contributors (https://konveyor.io/)
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+/// <reference types="cypress" />
+
+import {
+    login,
+    clickByText,
+    applySearchFilter,
+    preservecookies,
+    deleteByList,
+    createMultipleMigrationWaves,
+} from "../../../../utils/utils";
+import { button, name, clearAllFilters, SEC } from "../../../types/constants";
+
+import * as data from "../../../../utils/data_utils";
+import { MigrationWave } from "../../../models/migration/migration-waves/migration-wave";
+
+let migrationWavesList: Array<MigrationWave> = [];
+let invalidSearchInput = String(data.getRandomNumber());
+
+describe(["@tier2"], "Migration waves filter validations", function () {
+    before("Login and Create Test Data", function () {
+        // Peform login
+        login();
+
+        // Create multiple migration waves
+        migrationWavesList = createMultipleMigrationWaves(3);
+    });
+
+    after("Perform test data clean up", function () {
+        // Delete the business services
+        deleteByList(migrationWavesList);
+    });
+
+    beforeEach("Persist session", function () {
+        // Save the session and token cookie for maintaining one login session
+        preservecookies();
+    });
+
+    it("Name filter validations", function () {
+        MigrationWave.open();
+
+        // Enter an existing display name substring and assert
+        let validSearchInput = migrationWavesList[0].name.substring(0, 3);
```

Comment:
If you don't re-assign this variable, it can be a constant. `const validSearchInput =`